source/components/index.rst

```diff
@@ -57,6 +57,17 @@ The following TextDecorations are available:
 * Underlined
 * Obfuscated
 
+Additionally, you can set a font contained within the client's texture pack or within
+their Minecraft jar at ``assets\minecraft\font``.
+
+.. code:: java
+
+   // Creates a line of text in the uniform font, if applicable
+   final TextComponent textComponent3 = Component.text("This is fancy font")
+     .font(Key.key("uniform"))
+     .build();
```

Comment:
This code doesn't actually compile :p

Reply:
Blaming https://discord.com/channels/319355592605040642/923037274176913409/962815157694779423